vscode-trace-extension/src/trace-explorer/trace-tree.ts

```diff
@@ -166,7 +166,7 @@ export const fileHandler =
                     return;
                 }
 
-                const uri: string = resolvedTraceURI.path;
+                const uri: string = resolvedTraceURI.fsPath;
```

Comment:
Line 178: const name = uri.substring(uri.lastIndexOf('/') + 1);

That should be basename as well
Also variable should not be called uri, its more like "filePath"

Reply:
Fixed in latest commit.